docs/integrations.md

```diff
@@ -12,48 +12,49 @@ which supports fix actions, import sorting, and more.
 Ruff can be used as a [pre-commit](https://pre-commit.com) hook via [`ruff-pre-commit`](https://github.com/astral-sh/ruff-pre-commit):
 
 ```yaml
-# Run the Ruff linter.
 - repo: https://github.com/astral-sh/ruff-pre-commit
   # Ruff version.
-  rev: v0.0.291
+  rev: v0.1.4
   hooks:
+    # Run the linter.
     - id: ruff
-# Run the Ruff formatter.
-- repo: https://github.com/astral-sh/ruff-pre-commit
-  # Ruff version.
-  rev: v0.0.291
-  hooks:
+    # Run the formatter.
     - id: ruff-format
 ```
 
-To enable fixes, add the `--fix` argument to the linter:
+To enable lint fixes, add the `--fix` argument to the lint hook:
 
 ```yaml
-# Run the Ruff linter.
 - repo: https://github.com/astral-sh/ruff-pre-commit
   # Ruff version.
-  rev: v0.0.291
+  rev: v0.1.4
   hooks:
+    # Run the linter.
     - id: ruff
-      args: [ --fix, --exit-non-zero-on-fix ]
+      args: [ --fix ]
+    # Run the formatter.
+    - id: ruff-format
 ```
 
 To run the hooks over Jupyter Notebooks too, add `jupyter` to the list of allowed filetypes:
 
 ```yaml
-# Run the Ruff linter.
 - repo: https://github.com/astral-sh/ruff-pre-commit
   # Ruff version.
-  rev: v0.0.291
+  rev: v0.1.4
   hooks:
+    # Run the linter.
     - id: ruff
       types_or: [ python, pyi, jupyter ]
+      args: [ --fix ]
+    # Run the formatter.
+    - id: ruff-format
+      types_or: [ python, pyi, jupyter ]
 ```
 
-Ruff's lint hook should be placed after other formatting tools, such as Ruff's format hook, Black,
-or isort, _unless_ you enable autofix, in which case, Ruff's pre-commit hook should run _before_
-Black, isort, and other formatting tools, as Ruff's autofix behavior can output code changes that
-require reformatting.
+Ruff's lint hook should be placed run _before_ Ruff's formatter hook, as well as _before_ Black,
+isort, and other formatting tools, as Ruff's fix behavior can output code changes that require
+reformatting.
```

Comment:
@zanieb -- I'm a little nervous about this change, is there any scenario in which this would be wrong?

Reply:
Well.. if you're using format rules? Like `E501` would have false positives if you ran it before Black.

Reply:
But wouldn't Black then cause file changes, which would require you to re-run pre-commit? Or are there modes where people run pre-commit with allowed changes?

Reply:
Ruff would fail first, Black wouldn't run.